include/exadg/grid/grid_utilities.h

```diff
@@ -0,0 +1,252 @@
+/*
+ * grid_utilities.h
+ *
+ *  Created on: Jan 26, 2023
+ *      Author: fehn
+ */
```

Comment:
```suggestion
/*  ______________________________________________________________________
 *
 *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
 *
 *  Copyright (C) 2023 by the ExaDG authors
 *
 *  This program is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation, either version 3 of the License, or
 *  (at your option) any later version.
 *
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU General Public License for more details.
 *
 *  You should have received a copy of the GNU General Public License
 *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 *  ______________________________________________________________________
 */
```

Reply:
It seems that many of the files still have the old headers. Let's keep it like this and merge so that we can proceed with https://github.com/exadg/exadg/pull/323.